src/pages/register-a-beacon/additional-beacon-use.tsx

```diff
@@ -73,8 +73,9 @@ const onSuccessfulFormCallback: DestinationIfValidCallback = (
   const shouldCreateAdditionalUse =
     context.formData.additionalBeaconUse === "true";
   if (shouldCreateAdditionalUse) {
-    context.registration.createUse();
-    const useIndex = context.useIndex + 1;
+    const registration = context.registration;
+    registration.createUse();
+    const useIndex = registration.getRegistration().uses.length - 1;
```

Comment:
Fixes [this bug](https://trello.com/c/Zdc9Dbbq/471-bug-ensure-correct-beacon-use-index-is-used-when-creating-new) 🐛, yayy! 🎉